vacasa/connect/connect.py

```diff
@@ -229,6 +227,11 @@ def create_unit(self,
             "secured_by": secured_by,
         }
 
+        if code:
+            payload.update({"code": code})
+        if name:
+            payload.update({"name": name})
```

Comment:
```suggestion
            payload["name"] = name
```

Reply:
is this a performance thing or a more correct way of doing it?

Reply:
I doubt it really makes a noticeable difference from a performance perspective, it's more just a readability thing.  And even that is super minor.